franklin/builder/models.py

```diff
@@ -9,6 +9,9 @@
 from django.db import models
 
 from rest_framework import status
+from .queue import Queue
+
+from .queue import Queue
 
```

Comment:
Well A) I need to remove the duplicate, but B) this style of import is acceptable under [pep8](https://www.python.org/dev/peps/pep-0008/#imports). If we rather stick to the convention of no relative imports, that's fine as long as we all stick with it consistently.